definitions/variable/industry/production.yaml

```diff
@@ -0,0 +1,56 @@
+- Production|Iron and Steel|{Iron Commodity}:
+    description: Production of {Iron Commodity}
+    unit: Mt
+    tier: 1
+- Production|Iron and Steel|{Iron Commodity}|{Metal Origin}:
+    description: Production of {Iron Commodity} {Metal Origin}
+    unit: Mt
+    tier: 2
+- Production|Iron and Steel|{Iron Intermediate}:
+    description: Production of {Iron Intermediate}
+    unit: Mt
+    tier: 3
+
+- Production|Non-Metallic Minerals|{Non-Metallic Minerals Commodity}:
+    description: Production of {Non-Metallic Minerals Commodity}
+    unit: Mt
+    tier: 1
+- Production|Non-Metallic Minerals|{Non-Metallic Minerals Intermediate}:
+    description: Production of {Non-Metallic Minerals Commodity}
+    unit: Mt
+    tier: 1
+
+- Production|Non-Ferrous Metals|{Non-Ferrous Metals Commodity}:
+    description: Production of {Non-Ferrous Metals Commodity}
+    unit: Mt
+    tier: 1
+- Production|Non-Ferrous Metals|{Non-Ferrous Metals Commodity}|{Metal Origin}:
+    description: Production of {Non-Ferrous Metals Commodity} {Metal Origin}
+    unit: Mt
+    tier: 2
+- Production|Non-Ferrous Metals|{Non-Ferrous Intermediate}:
+    description: Production of {Non-Ferrous Intermediate}
+    unit: Mt
+    tier: 3
+
+- Production|Chemicals|{Chemicals Commodity}:
+    description: Production of {Chemicals Commodity}
+    unit: Mt
+    tier: 1
+- Production|Chemicals|{Primary Chemical}:
+    description: Production of {Primary Chemical}
+    unit: Mt
+    tier: 1
+- Production|Chemicals|{Chemicals Commodity}|{Metal Origin}:
```

Comment:
Does the distinction into primary and secondary make sense for chemicals?

Reply:
It is definitely needed for plastics. Primary chemicals can be produced from secondary feedstocks as well (e.g. chemical recycling of plastic waste), but currently this is not an established process in industry.

Reply:
So I'd only use it for plastics now and add it later for other chemicals if this is actually implemented by at least one model

Reply:
There is the chemical recycling route to plastics which essentially breaks the plastics up back into base chemicals which then avoids quality losses down the line (compared to mechanical recycling). So in principle having secondary label also combined with chemicals would make sense.